src/bus.c

```diff
@@ -56,3 +60,29 @@ char readBus(unsigned int addr, struct BUS * bus) {
 
   return data;
 }
+
+void writeFileBus(char * filePath, int offset, struct BUS * bus) {
```

Comment:
Will make sense rename this method?
Maybe something like `populateBus` or simply `writeProgramToBus`, what you think?

Reply:
👍

Reply:
I will do this change in the other Pr, to minimize merge problems ok?